test/methods.js

```diff
@@ -124,7 +124,32 @@ describe('nemo-view @methods@', function () {
       done();
     }, util.doneError(done));
   });
-
+  it('should return true using @ValidateText@Positive@ method', function (done) {
+    nemo.view.simple.pageHeaderValidateText('Sample form stuff').then(function (actual) {
```

Comment:
naming the argument `actual` doesn't indicate it is a boolean value. Please change to something like `isEqual`